### PR TITLE
OKTA-602037 Deprecate Log Streaming API on VuePress

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Deprecate Log Streaming API on VuePress site and redirect to Redocly site
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-602037](https://oktainc.atlassian.net/browse/OKTA-602037)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md](https://github.com/okta/okta-developer-docs/blob/98056bb2c013802f94d49c0f03a17bca620356c9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Flog-streaming%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/43e0e68d-77af-43c8-a0de-ce6e9e267392) | 8  | 100  | 100  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.e8fd7d99-d0b4-4ef4-a110-0cf9c90d19d3?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md](https://github.com/okta/okta-developer-docs/blob/2d4b220cc2862bba3701cff98cb37a5b58551f71/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Flog-streaming%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6b9c63f5-bd5f-4a10-9246-872823cc71bc) | 8  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/schemas/index.md](https://github.com/okta/okta-developer-docs/blob/2d4b220cc2862bba3701cff98cb37a5b58551f71/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fschemas%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a02b69ca-754b-4dd1-917e-81bdd7438ccb) | 73  | 71  | 64  | 80  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/2d4b220cc2862bba3701cff98cb37a5b58551f71/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2022-okta-identity-engine%2Findex.md) | [66](https://okta.acrolinx.cloud/api/v1/checking/scorecards/08abcfc0-a198-4ab0-ada3-8e43701f413a) | 35  | 70  | 43  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/release-notes/2022/index.md](https://github.com/okta/okta-developer-docs/blob/2d4b220cc2862bba3701cff98cb37a5b58551f71/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2022%2Findex.md) | [66](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c11794f5-e3a8-41ff-9080-0e08698b65f2) | 36  | 70  | 43  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/2d4b220cc2862bba3701cff98cb37a5b58551f71/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | [68](https://okta.acrolinx.cloud/api/v1/checking/scorecards/77e12b2b-9b91-4dfc-bfc4-3409324d2128) | 39  | 79  | 47  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/2d4b220cc2862bba3701cff98cb37a5b58551f71/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | [69](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5d61b0b8-2988-4dc7-8047-d2bed7822cc5) | 44  | 82  | 45  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.25267678-4505-44c6-ba85-8faa8087d903?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by isaacdurham-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/schemas/index.md [hunk 1]
```diff
@@ -1512,7 +1512,7 @@ The following response is only a subset of properties for brevity.
 
 Fetches the schema for a Log Stream type. The `${typeId}` element in the URL specifies the Log Stream type, which is either `aws_eventbridge` or `splunk_cloud_logstreaming`. Use the `aws_eventbridge` literal to retrieve the AWS EventBridge type schema, and use the `splunk_cloud_logstreaming` literal retrieve the Splunk Cloud type schema.
 
-See [Log Streaming API](/docs/reference/api/log-streaming) for examples of Log Stream objects.
+See [Log Streaming API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/LogStream/) for examples of Log Stream objects.
```
Comment (tomaszwrobel-okta) [inline packages/@okta/vuepress-site/docs/reference/api/schemas/index.md:1515]: Are we going to be adding separate OpenAPI docs for schemas too?

Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/reference/api/schemas/index.md:1515]: Yes, the Schemas API is separate: https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Schema/ (it's not complete and will need lots of work to get it up-to-date)

**Review by tomaszwrobel-okta — APPROVED** [review]

**Pull request merged**